File: core/client/game/src/commonGame/logic/team/PlayerTeam.cs


/// <summary>
/// 玩家队伍
/// </summary>
public class PlayerTeam:PlayerRoleGroup
{
	/** 创建队伍简版数据 */
	protected override RoleGroupSimpleData toCreateRoleGroupSimpleData()
	{
		return new TeamSimpleData();
	}
}

File: core/server/clientProject/commonClient/src/main/java/com/home/commonClient/logic/team/PlayerTeam.java
package com.home.commonClient.logic.team;

import com.home.commonBase.data.social.roleGroup.RoleGroupSimpleData;
import com.home.commonBase.data.social.team.TeamSimpleData;
import com.home.commonClient.logic.func.PlayerRoleGroup;

public class PlayerTeam extends PlayerRoleGroup
{
	/** 创建公会简版数据 */
	@Override
	protected RoleGroupSimpleData toCreateRoleGroupSimpleData()
	{
		return new TeamSimpleData();
	}
}
